eng/Versions.props

```diff
@@ -256,26 +256,28 @@
-    <MicrosoftNETRuntimeEmscripten3134Sdkwinx64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Sdkwinx64Version>
-    <MicrosoftNETRuntimeEmscripten3134Sdkwinarm64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Sdkwinarm64Version>
-    <MicrosoftNETRuntimeEmscripten3134Sdklinuxx64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Sdklinuxx64Version>
-    <MicrosoftNETRuntimeEmscripten3134Sdklinuxarm64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Sdklinuxarm64Version>
-    <MicrosoftNETRuntimeEmscripten3134Cachelinuxx64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Cachelinuxx64Version>
-    <MicrosoftNETRuntimeEmscripten3134Cachelinuxarm64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Cachelinuxarm64Version>
-    <MicrosoftNETRuntimeEmscripten3134Cacheosxx64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Cacheosxx64Version>
-    <MicrosoftNETRuntimeEmscripten3134Cacheosxarm64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Cacheosxarm64Version>
-    <MicrosoftNETRuntimeEmscripten3134Cachewinx64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Cachewinx64Version>
-    <MicrosoftNETRuntimeEmscripten3134Cachewinarm64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Cachewinarm64Version>
-    <runtimelinuxarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</runtimelinuxarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
-    <runtimelinuxmuslarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</runtimelinuxmuslarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
-    <runtimelinuxx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</runtimelinuxx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
-    <runtimelinuxmuslx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</runtimelinuxmuslx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
-    <runtimeosxarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</runtimeosxarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
-    <runtimeosxx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</runtimeosxx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
-    <runtimewinarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</runtimewinarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
-    <runtimewinx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</runtimewinx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
+    <MicrosoftNETRuntimeEmscripten3134Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Version>
+    <MicrosoftNETCoreRuntimeWasmNodeTransportVersion>9.0.0-alpha.1.24175.1</MicrosoftNETCoreRuntimeWasmNodeTransportVersion>
```

Comment:
Just curious, what is the scenario when these versions won't match? Are we "sometimes" producing only some packages?

Reply:
@steveisok ^

Reply:
Good question. The node transport pack comes out of dotnet/node and so it's not guaranteed to be the same. I also don't think we need to build it all the time.

Reply:
There will in practice only ever be one version emsdk and one for the node transfer package so I'd suggest setting those here doing something like
```xml
            <NodePkgVersion>$(runtimelinuxarm64MicrosoftNETCoreRuntimeWasmNodeTransportVersion)</NodePkgVersion>
            <EmsdkSdkVersion>$(MicrosoftNETRuntimeEmscripten3134Sdklinuxarm64Version)</EmsdkSdkVersion>
```
here and removing the platform based logic in mono.proj

Reply:
It is not required in the same way